pages/install/conda-install.rst

```diff
@@ -45,32 +47,87 @@ obtained are specified by adding the ``--channel`` option in the ``install`` or
 ``cantera`` channel, so we add ``--channel cantera`` and to tell Conda to look at the
 ``cantera`` channel in addition to the default channels.
 
-To use the scripts and modules installed in the ``spam`` environment,
-you must activate it it by running::
+To use the scripts and modules installed in the ``ct-env`` environment, including Jupyter,
+you must activate it it by running:
+
+.. code:: shell
+
+   conda activate ct-env
+
+**Option 2: Create a new environment using an environment file**
+
+This option is similar to **Option 1** but includes a few other packages that
+you may find helpful as you're working with Cantera. Copy and paste the contents
+of the file shown below into a file called ``environment.yaml``. Then, save the
+the file somewhere and remember that location.
+
+.. code:: yaml
+
+   name: ct-env
+   channels:
+   - cantera
+   - defaults
+   dependencies:
+   - python
+   - cantera
+   - ipython  # optional (needed for nicer interactive command line)
+   - jupyter  # optional (needed for Jupyter Notebook)
+   - matplotlib  # optional (needed for plots)
```

Comment:
`numpy` and `ruamel.yaml` are missing.

Reply:
They are automatically brought in with Cantera. I believe the conda recipes depend on `ruamel_yaml`, not `ruamel.yaml` (IIRC due to some namespace packaging issues in conda. There was discussion between Anton and the conda devs a few years ago leading to the different packages yadda yadda yadda). We end up being able to use either with all the `try..except ImportError`, but there's no need to double-specify, IMO.